rest-api/pom.xml

```diff
@@ -22,6 +22,18 @@
 
 	<build>
 		<plugins>
+			<plugin>
+				<artifactId>maven-jar-plugin</artifactId>
+				<version>3.0.2</version>
+				<configuration>
+					<archive>
+						<manifest>
+							<addDefaultImplementationEntries>true</addDefaultImplementationEntries>
```

Comment:
Does this use the current version set in the POM for Implementation-Version?

Reply:
Yep

Reply:
OK.  We need to add instructions somewhere, then to ensure that the POM version is incremented.  We will want to match the release tags to the POM version.  v0.21-beta is the current tag.

Reply:
Where do you want to document that? The readme? The pull request checklist?

Reply:
Actually... perhaps it would work better if the person creating the tag updates all the POM versions first.  That way we know the POM version is in sync.